client/src/components/EmailIcon.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import EIcon from "../assets/icons/iconfinder_icon-email_211660.svg";
+import styled from "styled-components";
+
+
+const EmailIcon = ({profile}) => {
+
+  const choseMailAdress = ()=>{
+     if(profile&&profile.email_adress){
+       return profile.email_adress
+     } else {
+       return 'contact@codeyourfuture.io'
+     }
+  }
+  return (
+    <>
+      <Container>
+        <ELink href={`mailto:${choseMailAdress()}`} target="blank">
```

Comment:
You could probably get rid of `choseMailAdress` and just do this instead:

```suggestion
        <ELink href={`mailto:${profile.email_adress || contact@codeyourfuture.io}`} target="blank">
```

Reply:
Also just a note, it should be "ad**d**ress" instead of "adress".